SPECS/libselinux/libselinux.spec

```diff
@@ -11,11 +11,11 @@ Source0:        https://github.com/SELinuxProject/selinux/releases/download/%{ve
 BuildRequires:  libsepol-devel
 BuildRequires:  pcre2-devel
 BuildRequires:  swig
-BuildRequires:  python3
 BuildRequires:  python3-devel
 BuildRequires:  python3-pip
 BuildRequires:  python3-setuptools
-BuildRequires:  python3-wheel
+# python3-wheel (non-toolchain package) cannot be added as a BR to libselinux (since it is a toolchain package)
+#BuildRequires:  python3-wheel
```

Comment:
Why leave the comment? In the end the version of the package we distribute is the one built without `python3-wheel`.

Reply:
To prevent someone from adding this again. Python3-wheel appears to really be required, although it is provided by the toolchain environment already.